README.md

```diff
@@ -48,16 +52,22 @@ NODE_ENV=test
 
 ## Installation
 
-To install all the project's dependencies run:
+If you don't have yarn installed, make sure you have a package manager like npm installed first, and then install yarn:
```

Comment:
I'm not sure if it's a good idea to document this as the way to install yarn: https://yarnpkg.com/getting-started/install. 
https://classic.yarnpkg.com/lang/en/docs/install/#windows-stable

Reply:
yeah this is new for yarn 2+ I think I'm still using yarn 1, I don't think it makes a difference but we can definitely include this yarn install link.